### DISCUSSION TITLE
I'd like to add form-fields api of QUploader without file_name.file_type in Form Data

### DISCUSSION THREAD

**Discussion by bombxdev** [quasarframework/quasar#13720] [General - Components / Directives / etc] [answered]
Hi teams, I'm using q-uploader. Now I'm trying to use `form-fields` but it's automatically add `file_name.jpg` such as `abcd.jpg` as the attach photo into one of parameters in `Form Data`. Then I got `{"statusCode":400,"message":"Unexpected field","error":"Bad Request"}` because I don't have that parameter in my backend function. How can I remove it from Form Data ? And I have no idea why the percentage of uploading fail at 99.99%?
![image](https://user-images.githubusercontent.com/62200480/174224225-83b32bde-e7aa-4972-addd-765889a44b93.png)
This is my code:
```
<q-uploader url="http://localhost:3000/api/upload/" style="max-width: 300px" method="POST"
          :form-fields="files => [{ name: 'orderId', value: '000' }, { name: 'file', value: files[0] }]"
          label="Upload Image" accept=".jpg, image/*" @added="file_selected" @start="start" @rejected="onRejected"
          @fail="onFailed" @uploaded="onUploaded" max-files="1" :readonly="isReadOnly" />
```
 Anyone, help me please.

**Comment by pratik227** [comment, marked as answer]
Hi @bombxdev 

You need to use `field-name`.

`:field-name="(file) => 'file'"`

Codepen - https://codepen.io/Pratik__007/pen/WNMmaBP

**Reply by bombxdev** [reply]
Thank you @pratik227 to guide me.  You made my day! \0.0/
This is my latest code : 
```
<q-uploader url="http://localhost:3000/api/upload/" style="max-width: 300px" method="POST"
          :form-fields="[{ name: 'orderId', value: '000' }]" field-name="file" label="Upload Image"
          accept=".jpg, image/*" @added="file_selected" @start="start" @rejected="onRejected" @fail="onFailed"
          @uploaded="onUploaded" max-files="1" :readonly="isReadOnly" />
```
I just realized `filed-name` means the name of field which I pass to my backend parameter api.
Everything works well now.
